odoo/addons/base/models/res_partner.py

```diff
@@ -389,7 +389,7 @@ def _compute_lang(self):
         """ While creating / updating child contact, take the parent lang by
         default if any. 0therwise, fallback to default context / DB lang """
         for partner in self.filtered('parent_id'):
-            partner.lang = partner.parent_id.lang or self.default_get(['lang'])['lang'] or self.env.lang
+            partner.lang = partner.parent_id.lang or self.default_get(['lang']).get('lang') or self.env.lang
```

Comment:
LGTM as a fix.

But could confirm the behavior of the mothod is expected?
When `self.default_get(['lang']).get('lang') is None`
new created `partner` will have `partner.lang == env.lang` if `parent_id` is specified, but `partner.lang==False` if `parent_id` is not specified.

The behavior seems not strictly consistent with its doc 🤔

Reply:
@tde-banana-odoo could you clarify what's intended here ? Thanks!

Reply:
Purpose was to remove overrides and onchange to have a complete compute. But default_get when there is an ir default always takes priority on compute field (because it is called in create, then field has a default value, and compute itself is not called as it is an editable computed field), which is really annoying. Hence the modification.

For the docstring, it is not the main thing you should look at. I mainly looked at tests and various try-and-die. Never trust docstrings, especially in code that comes from 10 year old code (old implementation of lang).

Cheers !